src/scripts/import-open-api.ts

```diff
@@ -782,43 +924,54 @@ const importOpenApi = async ({
   Object.entries(specs.paths).forEach(([route, verbs]: [string, PathItemObject]) => {
     Object.entries(verbs).forEach(([verb, operation]: [string, OperationObject]) => {
       if (["get", "post", "patch", "put", "delete"].includes(verb)) {
-        output += generateRestfulComponent(
-          operation,
-          verb,
-          route,
-          operationIds,
-          verbs.parameters,
-          specs.components,
-          customProps,
-          customGenerator,
-        );
+        if (ts) {
+          output += generateTypescriptApi(
+            verb,
+            operation.responses,
+            route,
+            operation.operationId,
+            operation.requestBody,
+            operation.parameters,
+          );
+        } else {
+          output += generateRestfulComponent(
+            operation,
+            verb,
+            route,
+            operationIds,
+            verbs.parameters,
+            specs.components,
+            customProps,
+            customGenerator,
+          );
+        }
       }
     });
   });
 
-  const haveGet = Boolean(output.match(/<Get</));
-  const haveMutate = Boolean(output.match(/<Mutate</));
-  const havePoll = Boolean(output.match(/<Poll</));
+  if (!ts) {
```

Comment:
This `if` should be useless, since we check than `<Get` is part of the generated code 😉